src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/ConfigService.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.service
+
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.visitscheduler.repository.PrisonRepository
+
+@Service
+@Transactional
+class ConfigService(
+  private val prisonRepository: PrisonRepository,
+) {
+
+  fun getSupportedPrisons(): List<String> {
```

Comment:
we could add the  @Transactional(readOnly = true) annotation to this method

Reply:
Good call